assets/js/stripe.js

```diff
@@ -264,6 +264,20 @@ jQuery( function( $ ) {
 			// Listen for hash changes in order to handle payment intents
 			window.addEventListener( 'hashchange', wc_stripe_form.onHashChange );
 			wc_stripe_form.maybeConfirmIntent();
+
+			//Mask CPF/CNPJ field when using Boleto
+			$( document ).on( 'change', '.woocommerce-checkout-payment', function() {
+				var TaxIdMaskBehavior = function (val) {
+						return val.replace(/\D/g, '').length >= 12 ? '00.000.000/0000-00' : '000.000.000-009';
+					},
+					spOptions = {
+						onKeyPress: function(val, e, field, options) {
+							field.mask(TaxIdMaskBehavior.apply({}, arguments), options);
+						}
+					};
+
+				$( '#stripe_boleto_tax_id' ).mask( TaxIdMaskBehavior, spOptions );
+			});
```

Comment:
I can't seem to add a CNPJ here. The mask doesn't let me do it.

Also, we need to normalize this code with the WP coding standards. More specifically, we're missing spaces before/after parenthesis.

It's unfortunate that the Stripe elements API doesn't provide a CPF tax ID form field, so we have to write our own and include the mask dependency.

Reply:
It seems like if you type the CNPJ it works but if you try to paste it, it does not work. I'll take a look at it

Reply:
fixed

Reply:
Let me know when you push the change, so I can re-review with the applied fixes.

It also helps a lot when you reference the hash on the "Fixed/resolve conversation" reply.